### DISCUSSION TITLE
How to define coil direction?

### DISCUSSION THREAD

**Discussion by QingfangLiu** [simnibs/simnibs#425] [8. Simulation workflows]
I’m new to SimNIBS, and I have a question that might be basic, but I haven’t found a clear answer and would appreciate some guidance. I’m using Localite to run TMS with subject-specific coordinates. In Localite, I haven’t manually adjusted the orientation of the coil and am relying on the system’s calculated entry points for each subject. Given this setup, how should I define the coil direction? Specifically, I’m confused about why adding 10 to the y-axis (as shown in some examples) is necessary. Can someone explain the reasoning behind this? Or how can I adjust this for my case?
I can provide more details about the experiment if needed. Thank you!

**Comment by bjornsig** [comment]
You define direction by creating another point relative to the target point. You can see a discussion on it here 
https://github.com/simnibs/simnibs/discussions/321
